### PR TITLE
PvConf#3 - feat: UI for preview configuration editor

### PR DESCRIPTION
**Pull request by devcatalin**
This PR...

## Changes

-

## Fixes

-

## How to test it

-

## Screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by olensmar — APPROVED** [review]
LGTM!

File: src/components/organisms/PreviewConfigurationEditor/PreviewConfigurationEditor.tsx [hunk 1]
```diff
@@ -1,5 +1,88 @@
...
+        <S.Label>Name your configuration:</S.Label>
+        <Input placeholder="Enter the configuration name" />
+      </S.Field>
+      <S.Field>
+        <S.Label style={{marginBottom: 0}}>Select which values files to use:</S.Label>
+        <S.Description>Drag and drop to specify order</S.Description>
+        <OrderedList items={valuesFileItems} onChange={setValuesFileItems} />
+      </S.Field>
+      <S.Field>
+        <S.Label>Select which helm command to use for this Preview:</S.Label>
```
Comment (olensmar) [inline src/components/organisms/PreviewConfigurationEditor/PreviewConfigurationEditor.tsx:60]: Add a help link going to corresponding Helm documentation?

File: src/constants/helmOptions.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+const helmCommonOptions = {
```
Comment (olensmar) [inline src/constants/helmOptions.ts:1]: could we add the description for each of these? or just have a link in the UI that takes the user to the Helm documentation?

Comment (devcatalin) [reply src/constants/helmOptions.ts:1]: We can add these later, for now we don't have the design to include the description for each option

File: src/components/organisms/PreviewConfigurationEditor/PreviewConfigurationEditor.tsx (hunk 1 above)
Comment (devcatalin) [reply src/components/organisms/PreviewConfigurationEditor/PreviewConfigurationEditor.tsx:60]: Sure, I'll add it

**Pull request merged**